benchmarks/.gitignore

```diff
@@ -0,0 +1 @@
+benches/datasets_paths.rs
```

Comment:
Shouldn't we also ignore the `target` folder and the `Cargo.lock` here?

Reply:
Since I added the `benchmarks` directory to the workspace defined in `milli/Cargo.toml` it shares it's `target` and `Cargo.lock` with `milli`, `http-ui`, etc in `milli/target` / `milli/Cargo.lock` (and these two are already ignored by `milli/.gitignore`

Reply:
Ho, yeah but why do we have a new gitignore here? Why don't we have this line in the current gitignore?

Reply:
I just thought it would be “easier” to maintain; If one day we decide to delete the `benchmarks` its `.gitignore` get deleted with it.

But honestly, it's as you want. I don't have any opinion on the subject.

Reply:
That's better indeed :)